woocommerce/woocommerce
# Move Template implementation classes to internal namespace

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:




Move Template implementation classes to internal namespace

Closes https://github.com/woocommerce/woocommerce/issues/40690 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Check that build is passing.
2. Smoke test: enable the new product editor at /wp-admin/admin.php?page=wc-settings&tab=advanced&section=features 
3. Go to Products > Add New
4. Check that the editor open successfully



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
